src/d3d9/d3d9_device.cpp

```diff
@@ -3836,7 +3836,11 @@ namespace dxvk {
     DxvkDeviceFeatures enabled = {};
 
     // Geometry shaders are used for some meta ops
+#ifdef DXVK_PLATFORM_DARWIN
+    enabled.core.features.geometryShader = supported.core.features.geometryShader;
```

Comment:
We only use this for swvp ProcessVertices so I dont mind just doing this always.

Reply:
Done, as a sidenote it seems that bump/normal mapping doesn't work at MacOS in M1 when testing the game but Im not sure if its because of MoltenVK side or because we don't have these 2 features available... anyway is not important as I disabled in game options when targeting this platform.